bluesky.py

```diff
@@ -0,0 +1,90 @@
+import models
+from granary import bluesky as gr_bluesky
+from oauth_dropins import bluesky as oauth_bluesky
+from flask_app import app
+import util
+import logging
+from flask import flash, render_template
+from oauth_dropins.webutil.util import json_loads
+from urllib.parse import quote
+
+logger = logging.getLogger(__name__)
+
+
+class Bluesky(models.Source):
+  """
+  A Bluesky account.
+  """
+  SHORT_NAME = 'bluesky'
+  GR_CLASS = gr_bluesky.Bluesky
+  OAUTH_START = oauth_bluesky.Start
+  AUTH_MODEL = oauth_bluesky.BlueskyAuth
+  URL_CANONICALIZER = util.UrlCanonicalizer(
+          domain=GR_CLASS.DOMAIN,
```

Comment:
this makes me think we need to support multiple domains in UrlCanonicalizer, so we can canonicalize staging.bsky.app => bsky.app. Obviously not needed in this PR though!

Reply:
Ugh yeah federation/multiple PDSs is just not something I've thought about yet, likely to be a headache but hoping we can use a lot of the same stuff Mastodon does?

Reply:
Oh this isn't federation, that's just the main official PSS's old hostname, there are still syndication links to it out there in the wild, eg on my site 😁

Agreed though, let's definitely punt on federation itself for now!